src/lib/Canvas.js

```diff
@@ -213,16 +213,36 @@ Canvas.prototype = {
         this.stopResizing();
     },
 
+    divBoundingClientRect: function() {
+        // Make sure our canvas has integral dimensions
+        var size = this.div.getBoundingClientRect();
+        var top = Math.floor(rect.top),
+            left = Math.floor(rect.left),
+            width = Math.ceil(rect.width),
+            height = Math.ceil(rect.height);
+
+        return {
+            top: top,
+            right: left+width,
+            bottom: top+height,
+            left: left,
+            width: width,
+            height: height,
+            x: rect.x,
+            y: rect.y
+        };
+    },
+
     checksize: function() {
         //this is expensive lets do it at some modulo
-        var sizeNow = this.div.getBoundingClientRect();
+        var sizeNow = this.divBoundingClientRect();
```

Comment:
I'd prefer methods names to be verbal, please.
So in this case prefix with `get` or `calc` , _e.g.,_ `getDivRect`